### ISSUE TITLE
"More" dots overlap search icon in main tool bar

### ISSUE THREAD

**Issue by arnowelzel** [nextcloud/server#24539]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Resize the browser window so that the main tool bar displays a menu overflow (. . . ) and look what happens when you slowly change the window size

### Expected behaviour
The overflow should never overlap the search icon.

### Actual behaviour
The overflow may overlap the search icon.

![grafik](https://user-images.githubusercontent.com/6613614/101096277-a2208d00-35bf-11eb-9b9d-29633e265774.png)

### Server configuration

**Operating system:**

**Web server:**

Apache 2.4

**Database:**

MariaDB

**PHP version:**

7.4

**Nextcloud version:**

20.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:**

Updated

**Where did you install Nextcloud from:**

nextcloud.com

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.6.0
  - activity: 2.13.3
  - admin_audit: 1.10.0
  - apporder: 0.11.0
  - bruteforcesettings: 2.0.1
  - calendar: 2.1.2
  - cloud_federation_api: 1.3.0
  - comments: 1.10.0
  - contacts: 3.4.2
  - contactsinteraction: 1.1.0
  - dashboard: 7.0.0
  - dav: 1.16.1
  - deck: 1.2.2
  - drawio: 0.9.8
  - extract: 1.2.5
  - federatedfilesharing: 1.10.1
  - federation: 1.10.1
  - files: 1.15.0
  - files_downloadactivity: 1.9.0
  - files_mindmap: 0.0.23
  - files_pdfviewer: 2.0.1
  - files_rightclick: 0.17.0
  - files_sharing: 1.12.0
  - files_texteditor: 2.14.0
  - files_trashbin: 1.10.1
  - files_versions: 1.13.0
  - files_videoplayer: 1.9.0
  - firstrunwizard: 2.9.0
  - forms: 2.0.4
  - gpxpod: 4.2.4
  - issuetemplate: 0.7.0
  - keeweb: 0.6.4
  - logreader: 2.5.0
  - lookup_server_connector: 1.8.0
  - nextcloud_announcements: 1.9.0
  - notes: 4.0.1
  - notifications: 2.8.0
  - oauth2: 1.8.0
  - onlyoffice: 6.1.0
  - password_policy: 1.10.1
  - phonetrack: 0.6.5
  - photos: 1.2.0
  - privacy: 1.4.0
  - provisioning_api: 1.10.0
  - quota_warning: 1.9.1
  - serverinfo: 1.10.0
  - settings: 1.2.0
  - sharebymail: 1.10.0
  - socialsharing_email: 2.1.0
  - spreed: 10.0.3
  - support: 1.3.0
  - survey_client: 1.8.0
  - systemtags: 1.10.0
  - tasks: 0.13.6
  - text: 3.1.0
  - theming: 1.11.0
  - theming_customcss: 1.7.0
  - twofactor_backupcodes: 1.9.0
  - updatenotification: 1.10.0
  - user_status: 1.0.1
  - viewer: 1.4.0
  - weather_status: 1.0.0
  - workflowengine: 2.2.0
Disabled:
  - encryption
  - files_external
  - files_markdown
  - recommendations
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud.0x0c.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "skeletondirectory": "",
        "overwrite.cli.url": "https:\/\/nextcloud.0x0c.de",
        "dbtype": "mysql",
        "version": "20.0.2.2",
        "installed": true,
        "htaccess.RewriteBase": "\/",
        "maintenance": false,
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "knowledgebaseenabled": false,
        "theme": "",
        "simpleSignUpLink.shown": false,
        "loglevel": 2,
        "log_authfailip": true,
        "mysql.utf8mb4": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "sendmail",
        "mail_smtpauthtype": "LOGIN",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "updater.release.channel": "stable",
        "trashbin_retention_obligation": "auto, 30",
        "app_install_overwrite": [
            "apporder",
            "deck",
            "keeweb"
        ],
        "has_rebuilt_cache": true,
        "onlyoffice": {
            "verify_peer_off": true
        }
    }
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:**

Firefox 83.0, Vivaldi 3.4.2066.106

**Operating system:**

Windows 10

**Comment by szaimen** [comment]
I can reproduce this on NC21.0.2 but it really only happens in edge cases.

**Comment by szaimen** [comment]
cc @nextcloud/designers

**Comment by juliushaertl** [comment]
Seems to happen only shortly before going over the mobile/desktop breakpoint at 1024px. The code in question is here https://github.com/nextcloud/server/blob/3e624dc39966f54c6d418eee2e8141fc5c21f751/core/src/init.js#L44-L65 in case anyone wants to investigate further.

**Comment by arnowelzel** [comment]
I think it would help to treat the "..." symbol as "app" if not all apps are visible.

**Comment by szaimen** [comment]
Hi, please update to 24.0.9 or better 25.0.3 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 25-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+

**Comment by arnowelzel** [comment]
Seems to be fixed at least in 25.0.3. However I did not test 24.0.9 as I don't use that version any longer.

**Comment by szaimen** [comment]
Thanks for verifying!
